prow/config.yaml

```diff
@@ -4929,6 +4930,48 @@ presubmits:
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
 
+  GoogleCloudPlatform/k8s-tpu-operator:
+  - name: pull-k8s-tpu-operator-unit
+    context: pull-k8s-tpu-operator-unit
+    agent: kubernetes
+    cluster: k8s-tpu-operator
+    always_run: true
+    rerun_command: /test pull-k8s-tpu-operator-unit
+    trigger: (?m)^/test( all| pull-k8s-tpu-operator-unit),?(\s+|$)
+    labels:
+      preset-service-account: "true"
+    skip_report: false
```

Comment:
This defaults to false. You can remove it here and below.

Reply:
Done.